packages/expo-image/src/Image.types.ts

```diff
@@ -437,6 +447,7 @@ export type ImageLoadEventData = {
     width: number;
     height: number;
     mediaType: string | null;
+    animated?: boolean | undefined;
```

Comment:
`undefined` is probably useless 🤔 

```suggestion
    isAnimated?: boolean;
```

Reply:
Yup!